rollup/internal/controller/relayer/l2_relayer.go

```diff
@@ -403,6 +404,10 @@ func (r *Layer2Relayer) ProcessPendingBatches() {
 
 		txHash, err := r.commitSender.SendTransaction(dbBatch.Hash, &r.cfg.RollupContractAddress, calldata, blob, fallbackGasLimit)
 		if err != nil {
+			if errors.Is(err, sender.ErrTooMuchBlobCarryingPendingTxs) {
```

Comment:
could add a rate limit metric here to find some clues in cases where `log.Debug` is not opened in rollup-relayer service.